sdk/identity/azure-identity/tests/test_win_vscode_credential.py

```diff
@@ -0,0 +1,75 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import sys
+import pytest
+from azure.core.credentials import AccessToken
+from azure.identity import CredentialUnavailableError
+try:
+    from unittest.mock import Mock
+except ImportError:  # python < 3.3
+    from mock import Mock  # type: ignore
+if sys.platform.startswith('win'):
+    from azure.identity._credentials.win_vscode_credential import (
+        WinVSCodeCredential,
+        _read_credential,
+        _cred_write,
+        _cred_delete,
+    )
+
+@pytest.mark.skipif(not sys.platform.startswith('win'), reason="This test only runs on Windows")
+def test_win_vscode_credential():
+    service_name = u"VS Code Azure"
```

Comment:
Why must all these ASCII literals be `unicode` on 2.7?

Reply:
This is not supposed to be used by user. I added the secret just for test purpose.

Reply:
Sure, but why do you explicitly make these literals `unicode` on 2.7?